src/sagemaker/fw_utils.py

```diff
@@ -117,6 +126,8 @@ def _is_dlc_version(framework, framework_version, py_version):
     """
     lowest_version_list = MERGED_FRAMEWORKS_LOWEST_VERSIONS.get(framework)
     if isinstance(lowest_version_list, dict):
+        if py_version not in lowest_version_list:
+            raise ValueError("{} is not supported in {}.".format(framework, py_version))
```

Comment:
how are we handling other cases where EI wouldn't work, e.g. PyTorch 1.2? I think it'd be better to keep this method strictly about determining if the version is one where a DLC image is used, and to handle other logic (e.g. "is this framework + Python version combination valid?") in its own location.

Reply:
The version safe guard is here: https://github.com/aws/sagemaker-python-sdk/blob/adbf8dbb9f29ee968bb8a8f819dad20545736ea5/src/sagemaker/fw_utils.py#L93

I've moved the framework_version + py_version check to framework model class and moved unit test accordingly

Reply:
sorry for not being clear - I meant that it didn't really fit in that particular function, but I think it's okay to leave in `fw_utils`. I'd probably modify this function: https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/fw_utils.py#L261-L263